careers/current-openings/index.html

```diff
@@ -0,0 +1,113 @@
+{% extends "layout-side-nav.html" %}
+{% import "_vars-careers.html" as vars with context %}
+{%- set active_nav_id = 'current-openings' -%}
+
+{% block title -%}
+    Current Openings
+{%- endblock %}
+
+{% block desc -%}
+    Information on careers at the CFPB.
+{%- endblock %}
+
+{% block css -%}
+    {{ super() }}
+    {# TODO: Resolve slick font issue.
+             For the slick stylesheet to pick up its font and img reference the
+             fonts and images found at /src/vendor/slick-carousel/slick/ should
+             be included in same directory as slick.css.
+             However, including the slick fonts makes the dots on the
+             /the-bureau/ page at mobile sizes enormous.
+    #}
+    <link rel="stylesheet" href="/static/css/slick.css">
+{%- endblock %}
+
+{% block content_modifiers -%}
+    {{ super() }}
+    careers-all-current-openings
+{%- endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }}
+    content__flush-bottom
+{%- endblock %}
+
+{% block content_main %}
+
+    {% import "_macro-table-current-openings.html" as table_current_openings %}
+
+    <section class="block
+                    block__flush-top
+                    block__sub">
+            <h1>Current Openings</h1>
+            <p class="h3">
+                Find the position that’s right for you. See positions by title, location, or closing date. We recommend applying at least a full day before the closing date, in case you have issues or questions you need answered.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/0081829f181bd6df229cb4d3f34c217665f5306d